src/chrome/content/rules/Nintendolife.com.xml

```diff
@@ -0,0 +1,26 @@
+<!--
+	Nonfunctional hosts in *.nintendolife.com:
+
+# All sites stay as http with this ruleset active.
```

Comment:
Can you explain what do you mean by this?

Reply:
All the sites stay as http:// none of them redirect to https://

Reply:
It's fine to remove the entire comment then.